src/main/java/org/jabref/cli/XmpUtilMain.java

```diff
@@ -0,0 +1,129 @@
+        if (bibFile.endsWith(".bib") && pdfFile.endsWith(".pdf")) {
+            try (FileReader reader = new FileReader(bibFile)) {
+                ParserResult result = new BibtexParser(importFormatPreferences, Globals.getFileUpdateMonitor()).parse(reader);
+                XmpUtilWriter.writeXmp(Paths.get(pdfFile), result.getDatabase().getEntries(), result.getDatabase(), xmpPreferences);
+                System.out.println("Metadata sucessfully written to Pdf.");
+            }
+        } else {
+            System.err.println("Insert correct file paths (.bib and .pdf)");
+        }
+    }
+
+    /**
+     * Print usage information for the console tool xmpUtil.
+     */
+    private static void printMenu() {
+        System.out.println("---------------------Menu-----------------------");
+        System.out.println("(0) Exit");
+        System.out.println("(1) Read metadata from PDF and print as bibtex");
+        System.out.println("(2) Write entries in bib file to Pdf metadata");
+        System.out.println("    To report bugs visit https://issues.jabref.org");
+        System.out.println("-------------------------------------------------");
+        System.out.print("Choose an option: ");
+    }
+
+    /**
+     * The tool is implemented as a console application with a read-evaluate-print cycle.
+     */
+    public static void executeXmpConsoleApplicaton() {
+        if (Globals.prefs == null) {
+            Globals.prefs = JabRefPreferences.getInstance();
+        }
+
+        xmpPreferences = Globals.prefs.getXMPPreferences();
+        importFormatPreferences = Globals.prefs.getImportFormatPreferences();
+
+        BufferedReader consoleReader = new BufferedReader(new InputStreamReader(System.in));
+
+        int option = -1;
+        while (option != 0) {
+            try {
+                XmpUtilMain.printMenu();
+                option = Integer.parseInt(consoleReader.readLine());
+
+                if (option == 0) {
+                    break;
+                } else if (option == 1) {
+                    System.out.print("Insert your filename (.pdf): ");
+                    String filename = consoleReader.readLine().trim();
+                    XmpUtilMain.readPdfAndPrintBib(filename);
+                } else if (option == 2) {
+                    System.out.print("Insert your filename (.bib): ");
+                    String bibFile = consoleReader.readLine().trim();
+                    System.out.print("Insert your filename (.pdf): ");
+                    String pdfFile = consoleReader.readLine().trim();
+                    XmpUtilMain.writeBibFileToPdfMetadata(bibFile, pdfFile);
+                }
+            } catch (IOException | TransformerException e) {
+                System.err.println(e.getMessage());
+            }
+        }
+    }
+
+    public static void main(String[] args) {
```

Comment:
Do we actually need this tool as a separate executable now that the functionality is included in the usual CLI?

Reply:
It's a convenient tool to add metadata to pdf files. If there is the possiblity to build a standalone jar with the multi-model build, why not use the chance to offer this tool to more users and make this tool more popular?